core/src/test/java/org/hibernate/ogm/backendtck/callbacks/PostUpdateTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.callbacks;
+
+import static org.hibernate.ogm.utils.GridDialectType.INFINISPAN;
+import static org.hibernate.ogm.utils.GridDialectType.INFINISPAN_REMOTE;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import javax.persistence.EntityManager;
+
+import org.hibernate.ogm.utils.SkipByGridDialect;
+import org.hibernate.ogm.utils.jpa.OgmJpaTestCase;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+@SkipByGridDialect(value = { INFINISPAN,INFINISPAN_REMOTE }, comment = "The dialect has strangenesses with 'postupdate' callback")
```

Comment:
This commet does not help at all in understanding what's happening with INFINISPAN